runtime/src/lib.rs

```diff
@@ -143,15 +143,20 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     transaction_version: 1,
 };
 
-pub const MILLISECS_PER_BLOCK: u64 = 6000;
+pub const MILLISECS_PER_BLOCK: u64 = 12000;
```

Comment:
why wouldn't we use the same blocktime as in the standalone chain of 4.32 seconds https://github.com/DataHighway-DHX/node/blob/master/pallets/primitives/src/constants/mod.rs#L28

Reply:
@ltfschoen I checked both Polkadot 0.9.9 Substrate standalone chain template and Substrate parachain template. Value is different in both of them. Please check here 

https://github.com/paritytech/substrate/blob/polkadot-v0.9.9/bin/node-template/runtime/src/lib.rs#L113

https://github.com/substrate-developer-hub/substrate-parachain-template/blob/polkadot-v0.9.9/runtime/src/lib.rs#L128

We followed  Substrate standalone chain template for our standalone chain and  Substrate parachain template for our parachain.

Reply:
see the internal DHX Halving & Mining spreadsheet that i sent you, we've done calculations based on a 4.32 second block time, and i believe we should be using this on both standalone chain and parachain unless decentralised governance votes to change it, as it would affect other things to

Reply:
added to #227